static/css/accounting.css

```diff
@@ -11,15 +11,21 @@ main.index .toctree-wrapper > .row:first-child > .col-md-3 {
 .doc-aside p{
     padding: .5rem;
 }
+.doc-aside div {
+    border: 1px solid rgba(255, 255, 255, 0);
```

Comment:
I'm not sure what is the point of this. This will put an **invisible** border (?) What are we trying to achieve ?

It happens a lot in the pr.

Reply:
Hi @rugo-odoo, the reason I added an 1px invisible border is to avoid the 1px layout shift that happens when the 1px black border appears on hover :)

Reply:
I believe much of this is unnecessary.

1) For the one with `.highlight-op`, we can just change the border color. We already have a border so it won't shift.

2) If I'm not mistake, to prevent the shift, we can just use `box-sizing: border-box` for the elements we need. [border-box](https://developer.mozilla.org/en-US/docs/Web/CSS/box-sizing) will not add any width/height with the border.

It will be cleaner and easier to maintain.